src/app/components/client/toolbar/UserMenu.tsx

```diff
@@ -62,6 +66,9 @@ export const UserMenu = (props: UserMenuProps) => {
       case itemKeys.settings:
         settingsItemRef.current?.click();
         break;
+      case itemKeys.contact:
+        contactItemRef.current?.click();
+        break;
```

Comment:
These lines seem to be not covered by the unit tests, yet. We’d need to test a click event on the contact item in [UserMenu.test.tsx](https://github.com/mozilla/blurts-server/blob/MNTOR-2880-contact-us/src/app/components/client/toolbar/UserMenu.test.tsx).

Reply:
Yep sorry, was end-of-day so I wasn't able to handle the CI failure right away. Fixed in e5a72884181c029d5eb1a9954cd0597aa406f41f.